templates/sso74-https.json

```diff
@@ -5,19 +5,19 @@
         "annotations": {
             "iconClass" : "icon-sso",
             "tags" : "sso,keycloak,jboss,hidden",
-            "version": "7",
-            "openshift.io/display-name": "Red Hat Single Sign-On CD 7 Tech Preview (Ephemeral with passthrough TLS)",
+            "version": "7.4.0.GA",
```

Comment:
`"v7.4.0.GA"` is preferred over `"7.4.0.GA"`. See [current 7.3 tags|https://github.com/jboss-container-images/redhat-sso-7-openshift-image/releases] IIRC the `vRelease` form is used in referenced documentation already, so would be good to be consistent

Reply:
Not needed (see related discussion), closing.